pci-compliance/index.md

```diff
@@ -4,7 +4,19 @@ title: PCI Compliance
 permalink: /pci-compliance/
 ---
 
-Is a thing.
+The [Payment Card Industry Data Security Standard](http://en.wikipedia.org/wiki/Payment_Card_Industry_Data_Security_Standard) is a set of rules developed by the major credit card issuers.
 
-- [SAQs (Self Assessment Questionnaires)](/pci-compliance/saqs)
-- [Integrations and their SAQs](/pci-compliance/integrations)
+If you have a data breach or process a large volume of transactions and do not abide by these rules, you may be fined.
```

Comment:
I swear something I was reading mentioned fines, I can try to find it again. Updated the PR with your suggestion.